Upstream-Status: Backport
Signed-off-by: Ross Burton <ross.burton@intel.com>

From bdaa9de44a197ba1297b990422ed9e515289f141 Mon Sep 17 00:00:00 2001
From: Julien Cristau <jcristau@debian.org>
Date: Tue, 1 Apr 2014 10:11:13 +0200
Subject: [PATCH 1/2] configure: fix build without xatracker

ACTION-IF-NOT-FOUND is the fourth argument to PKG_CHECK_MODULES, not the
fifth.

Debian bug#743239 <https://bugs.debian.org/743239>

Reviewed-by: Thomas Klausner <wiz@NetBSD.org>
Reviewed-by: Jakob Bornecrantz <jakob@vmware.com>
Signed-off-by: Julien Cristau <jcristau@debian.org>
Signed-off-by: Jakob Bornecrantz <jakob@vmware.com>
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index dd3ad4e..be40ed7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -126,7 +126,7 @@ if test x$BUILD_VMWGFX = xyes; then
 			  [PKG_CHECK_EXISTS([xatracker = 2.0.0],
 	                  [AC_DEFINE([HAVE_XA_2], 1,
                		  [Has version 2 of XA])])],
-			  [],[BUILD_VMWGFX=no])
+			  [BUILD_VMWGFX=no])
 #
 # Check for prime.
 #
-- 
1.7.10.4

